### DISCUSSION TITLE
OpenTelemetry.Instrumentation.AspNetCore AddAspNetCoreInstrumentation not working

### DISCUSSION THREAD

**Discussion by mcmugchu** [open-telemetry/opentelemetry-dotnet#4358] [Q&A]
Hello, I followed the example to add AspNetCoreInstrumentation and HttpClientInstrumentation into my .Net6 WebAPI project.

Services.AddOpenTelemetry()
               .ConfigureResource(configureResource)
               .WithTracing(builder =>
                {
                    builder
                        .AddSource(Instrumentation.ActivitySourceName)
                        .SetSampler(new AlwaysOnSampler())
                        .AddHttpClientInstrumentation()
                        .AddAspNetCoreInstrumentation()
                        .AddConsoleExporter();
                });

When I run the app and call the API, I can see the trace created by HttpClientInstrumentation when there's outgoing http traffic in my code, but there's no any trace created by AspNetCoreInstrumentation.
There was no any error message, is there any hint for me to troubleshoot? Thanks!

**Comment by utpilla** [comment]
Could you provide a minimal repro app which shows this behavior?
